service/src/main/resources/db/changesets/20220609_activity_log_table.yaml

```diff
@@ -0,0 +1,25 @@
+databaseChangeLog:
+- changeSet:
+    id: workspace activity log table
+    author: yuhuyoyo
+    changes:
+    - createTable:
+        tableName: workspace_activity_log
```

Comment:
Rename file to 20220609_workspace_activity_log_table.yaml

Reply:
since we are having one activity log table. keeping it as activity_log